### PR TITLE
Add preauthExpiresAt property to payment request

### PR DESCRIPTION
**Pull request by laurencemcknight**
As part of the preauth expiry story, a preAuthExpiresAt property was added to the Payment Request. This PR updates the docs reflect this change.

### CODE REVIEW

File: src/api/payment-requests/payment-requests.md [hunk 1]
```diff
@@ -64,26 +64,26 @@ version (documented on this page) and the "legacy" version (documented at
...
+| conditionsEnabled    | Boolean            | Flag to indicate that a merchant is able to accept [Payment Conditions](#payment-condition).                                                                             |
+| patronNotPresent     | Boolean            | Flag to indicate the patron is not physically present. This may affect payment conditions or available [Payment Options][].                                              |
+| cancellationReason   | String             | The reason that the payment request was cancelled. See [Cancellation Reasons](#cancellation-reasons) for possible values.                                                |
+| preAuth              | Boolean            | Flag to indicate the if the request is a pre authorization for supported [Asset Types][].                                                                                |
+| preAuthExpiresAt     | {% dt Timestamp %} | Pre authorization completions and releases will be accepted until this time.                                                                                             |
```
Comment (heymatthew) [inline src/api/payment-requests/payment-requests.md:86]: We need error codes if you cant perform these actions, and they need to be documented below this table somewhere.

Comment (heymatthew) [reply src/api/payment-requests/payment-requests.md:86]: I expect to see this new field in the examples that have preAuth: true too.

Comment (heymatthew) [reply src/api/payment-requests/payment-requests.md:86]: Please update this when you're done:
https://www.notion.so/centrapay/The-API-Protocols-for-Preauth-d2f6800d164b47928d9fee0838b7b655

Comment (heymatthew) [reply src/api/payment-requests/payment-requests.md:86]: Please update notion API internal documentation examples

Comment (heymatthew) [reply src/api/payment-requests/payment-requests.md:86]: Change "pre authorization" to "Pre Auth" everywhere.

File: src/api/payment-requests/payment-requests.md [hunk 2]
```diff
@@ -561,6 +560,53 @@ Payment Activities are created when a Payment Request has been **created**, **pa
...
+        "type": "farmlands.nzd.main",
+        "description": "Farmlands NZD",
+        "settlementDate": "2021-06-28T04:04:27.426Z",
+        "total": {
+          "amount": "1000",
+          "currency": "NZD"
+        }
+      }
+    ]
+  },
```
Comment (heymatthew) [inline src/api/payment-requests/payment-requests.md:604]: Remove this. Small as possible.

File: src/api/payment-requests/payment-requests.md [hunk 3]
```diff
@@ -561,6 +560,53 @@ Payment Activities are created when a Payment Request has been **created**, **pa
...
+  "barcode": 503901342,
+  "merchantId": "26d3Cp3rJmbMHnuNJmks2N",
+  "merchantName": "Centrapay Café",
+  "configId": "5efbe2fb96c08357bb2b9242",
+  "value": { "currency": "NZD", "amount": "1000" },
+  "paymentOptions": [
+    {
+      "amount": "1000",
+      "assetType": "centrapay.nzd.main"
+    },
```
Comment (heymatthew) [inline src/api/payment-requests/payment-requests.md:579]: Remove this. This option is not possible for preAuth.

File: src/api/payment-requests/payment-requests.md [hunk 4]
```diff
@@ -561,6 +560,53 @@ Payment Activities are created when a Payment Request has been **created**, **pa
 }
 {% endjson %}
 
+{% h4 Example response payload for a pre authorization Payment Request %}
+
+{% json %}
+{
+  "id": "MhocUmpxxmgdHjr7DgKoKw",
+  "shortCode": "CP-C7F-ZS5",
+  "url": "https://app.centrapay.com/pay/MhocUmpxxmgdHjr7DgKoKw",
```
Comment (heymatthew) [inline src/api/payment-requests/payment-requests.md:569]: Where did this come from? It's not described in the model.

Please follow up with the author who wrote this and get it fixed in another PR.

File: src/api/payment-requests/payment-requests.md [hunk 5]
```diff
@@ -561,6 +560,53 @@ Payment Activities are created when a Payment Request has been **created**, **pa
...
+      "amount": "1000",
+      "assetType": "centrapay.nzd.main"
+    },
+    {
+      "amount": "1000",
+      "assetType": "farmlands.nzd.main"
+    }
+  ],
+  "merchantConditions": [],
+  "status": "paid",
```
Comment (heymatthew) [inline src/api/payment-requests/payment-requests.md:586]: Status "New"

File: src/api/payment-requests/payment-requests.md [hunk 6]
```diff
@@ -935,9 +981,10 @@ When you call release on a preauth Payment Request any remaining funds that were
 
 {% h4 Error Responses %}
 
-| Status |                 Code                         |                     Description                                                              |
-| :----- | :------------------------------------------- | :------------------------------------------------------------------------------------------- |
-| 403    | {% break _ INVALID_PAYMENT_REQUEST_TYPE %}   | The Payment Request is not related to a pre authorization                                    |
+| Status |                    Code                    |                                          Description                                           |
+| :----- | :----------------------------------------- | :--------------------------------------------------------------------------------------------- |
+| 403    | {% break _ INVALID_PAYMENT_REQUEST_TYPE %} | The Payment Request is not related to a pre authorization                                      |
+| 403    | {% break _ PREAUTH_EXPIRED %}              | Attempted to release the pre authorization Payment Request after `preAuthExpiresAt` has passed |
```
Comment (heymatthew) [inline src/api/payment-requests/payment-requests.md:987]: > Attempted to release the pre authorization Payment Request after `preAuthExpiresAt` has passed

Too wordy. Short as possible please.

**Review by heymatthew — CHANGES_REQUESTED** [review]

File: src/api/payment-requests/payment-requests.md (hunk 4 above)
Comment (laurencemcknight) [reply src/api/payment-requests/payment-requests.md:569]: This field does exist and it is returned in our APIs however it is not documented in the model. [I have a seperate PR up that adds this field.](https://github.com/centrapay/centrapay-docs/pull/454)

**Review by heymatthew — APPROVED** [review]
LGTM 🚀 it!
